lib/evaluate/intrinsics.cc

```diff
@@ -976,14 +976,16 @@ std::optional<SpecificCall> IntrinsicInterface::Match(
        dummy[dummyArgPatterns].keyword != nullptr;
        ++dummyArgPatterns) {
   }
-  std::vector<ActualArgument *> actualForDummy(dummyArgPatterns, nullptr);
   // MAX and MIN (and others that map to them) allow their last argument to
   // be repeated indefinitely.  The actualForDummy vector is sized
   // and null-initialized to the non-repeated dummy argument count,
   // but additional actual argument pointers can be pushed on it
   // when this flag is set.
   bool repeatLastDummy{dummyArgPatterns > 0 &&
       dummy[dummyArgPatterns - 1].optionality == Optionality::repeats};
+  std::size_t nonRepeatedDummys{
```

Comment:
English quirk: the plural of "dummy" is "dummies".

Reply:
Thanks